omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentBytesResource1_10.java

```diff
@@ -37,7 +38,7 @@
 	protected final Log log = LogFactory.getLog(getClass());
 	
 	@RequestMapping(value = AttachmentsConstants.ATTACHMENT_BYTES_URI, method = RequestMethod.GET)
-	public void getFile(@PathVariable("uuid") String uuid, HttpServletResponse response) throws ResponseException {
+	public void getFile(@PathVariable("uuid") String uuid,@RequestParam(required = false, value = "view") String view, HttpServletResponse response) throws ResponseException {
```

Comment:
A detail but this:
```java
String uuid,@RequestParam(required =
```
so the missing blank space between "`uuid`" and "`@RequestParam`" shows that the file is misformatted. In other words you did not run `mvn clean package` before pushing.

Always run `mvn clean package` before pushing code to GitHub.

Reply:
Thanks for this.